cmd/scanner/scanner.go

```diff
@@ -119,35 +153,42 @@ func main() {
 	// Ignore Logger sync error
 	defer func() { _ = logger.Sync() }()
 
+	scan := NewScanner(dbURL, dbPass, credsFile, logger)
+	scan.logger.Info("Starting ClusterIQ Scanner",
+		zap.String("version", version),
+		zap.String("commit", commit),
+		zap.String("dbURL", dbURL),
+		zap.String("credentials file", credsFile),
+	)
+
+	var err error
```

Comment:
Maybe you don’t have to declare `err` here, you are initialized on line 166

Reply:
you're right. Var declaration removed